packages/server/worker/src/lib/utils/cache-handler.ts

```diff
@@ -29,6 +30,9 @@ const getCache = async (folderPath: string): Promise<CacheMap> => {
 export const cacheHandler = (folderPath: string) => {
     return {
         async cacheCheckState(cacheAlias: string): Promise<CacheState | string | undefined> {
+            if (isDev) {
```

Comment:
Remove CacheState and move it as local enum (non exported) to where it's used, as string is generic